src/views/List.jsx

```diff
@@ -6,12 +6,11 @@ export function List({ data }) {
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
+
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((data) => (
```

Comment:
To make this easier to read and avoid confusion, it's best to distinguish between the array (`data`) and its values (also called `data` here). 😊

Often the convention is going from plural to singular, for example, if the array is called `books`, then the item would be `book`. Here you could use something like `item`.